components/mbedtls/Kconfig

```diff
@@ -1,5 +1,27 @@
 menu "mbedTLS"
 
+config MBEDTLS_PLATFORM_MEMORY
+   bool "Enable mbedTLS memory allocation layer."
```

Comment:
Suggest "Enable custom mbedTLS memory allocation layer", so it's clear this is more than just having mbedTLS support memory allocation.